src/sentry/search/south_migrations/0002_auto__del_searchtoken__del_unique_searchtoken_document_field_token__de.py

```diff
@@ -57,10 +57,10 @@ def backwards(self, orm):
                     self.gf('sentry.db.models.fields.FlexibleForeignKey')(to=orm['sentry.Group'])
                 ), (
                     'date_changed',
-                    self.gf('django.db.models.fields.DateTimeField')(default=datetime.datetime.now)
```

Comment:
Why do all of these need to be removed?

Reply:
the defaults break on Django 1.8 since it cant evaluate them -- i didnt want to fix, and they were incorrect (its static datetimes) so I just removed them instead

Reply:
Yeah, I knew they were incorrect. :) Was this fixed for new migrations then so we don't keep producing bad values in new migrations? My eyes glossed over the South changes since I don't know much about South.